api/services/Usernotes.js

```diff
@@ -16,7 +17,7 @@ exports.addUsernote = async function (redToken, modname, subreddit, user, noteTe
   var mods = parsed.constants.users;
   var warnings = parsed.constants.warnings;
   var notes = decompress(parsed.blob);
-  if (!notes.user) {
+  if (!notes[user]) {
```

Comment:
This line was causing #503. The bugged version was looking for a key matching the literal string "user", instead of a key that matched the actual username.